ssl/ssl_locl.h

```diff
@@ -987,6 +988,11 @@ struct ssl_ctx_st {
     size_t (*record_padding_cb)(SSL *s, int type, size_t len, void *arg);
     void *record_padding_arg;
     size_t block_padding;
+
+    /* Session ticket appdata */
+    tls_generate_session_ticket_cb_fn generate_ticket_cb;
+    tls_decrypt_session_ticket_cb_fn decrypt_ticket_cb;
+    void* ticket_cb_data;
```

Comment:
(Personally, I prefer the asterisk next to the type (since it's a void pointer); but it's not OpenSSL style.)